cli/cmd/timescaledb/connect.go

```diff
@@ -24,46 +23,44 @@ var timescaledbConnectCmd = &cobra.Command{
 }
 
 func init() {
-	timescaledbCmd.AddCommand(timescaledbConnectCmd)
+	Cmd.AddCommand(timescaledbConnectCmd)
 	timescaledbConnectCmd.Flags().BoolP("master", "m", false, "directly execute session on master node")
 	timescaledbConnectCmd.Flags().StringP("dbname", "d", "postgres", "database name to connect to")
```

Comment:
can't we get this from deployment information. Even if we want the option to modify, the default dbname should be from the deployment info